src/library/coverartcache.cpp

```diff
@@ -7,9 +7,42 @@
 #include "library/coverartcache.h"
 #include "library/coverartutils.h"
 
-// Large cover art wastes space in our cache when we typicaly won't show them at
-// their full size. If no width is specified, this is the maximum width cap.
-const int kMaxCoverWidth = 300;
+
+namespace {
+    QString pixmapCacheKey(quint16 hash, int width) {
+        if (width == 0) {
+            return QString("CoverArtCache_%1").arg(QString::number(hash));
+        } else {
+            return QString("CoverArtCache_%1_%2")
+                    .arg(QString::number(hash)).arg(width);
+        }
+    }
+
+    // Large cover art wastes space in our cache when we typically won't show them at
+    // their full size. If no width is specified, this is the maximum width cap.
+    const int kMaxCoverSize = 300;
```

Comment:
That was the old name, which was not precise since it limit the height and width.
kMaxEdgeLength and kMaxHeightAndWidth would be correct.